vhdbuilder/packer/components.json

```diff
@@ -428,7 +428,8 @@
       "downloadLocation": "/opt/cni/downloads",
       "downloadURL": "https://acs-mirror.azureedge.net/cni-plugins/v*/binaries",
       "versions": [
-        "0.9.1"
+        "0.9.1",
```

Comment:
run_test will fail in the pipeline because 0.9.1 is not cached in the vhd any more according to the change in vhdbuilder/packer/install-dependencies.sh